core/arch/arm/include/mm/core_mmu.h

```diff
@@ -48,14 +48,12 @@
 
 /*
  * CORE_MMU_L1_TBL_OFFSET is used when switching to/from reduced kernel
- * mapping. The actual value depends on internals in core_mmu_lpae.c and
- * core_mmu_v7.c which we rather not expose here. There's a compile time
- * assertion to check that these magic numbers are correct.
+ * mapping. The actual value depends on internals in core_mmu_lpae.c which
+ * we rather not expose here. There's a compile time assertion to check
+ * that these magic numbers are correct.
  */
```

Comment:
minor: I suggest to move the inline comment inside `#ifdef CFG_WITH_LPAE`/`#endif`.

Reply:
OK